project/project_test.go

```diff
@@ -19,3 +20,27 @@ func TestEventEquality(t *testing.T) {
 		t.Fatal("Events match")
 	}
 }
+
+func TestBadFile(t *testing.T) {
```

Comment:
Could this be `TestParseWithBadFile()` so it's clear which function is being tested?